esp/src/src-react/components/controls/Chip.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import { styled } from '@fluentui/utilities';
+import { lighten, fade } from "@material-ui/core/styles";
+
+import { IStyleFunctionOrObject, classNamesFunction } from '@fluentui/react';
+import { IStyle, ITheme } from '@fluentui/style-utilities';
+
+import { StatusErrorFullIcon } from "@fluentui/react-icons-mdl2";
+
+interface IChipProps extends React.RefAttributes<HTMLDivElement> {
+    label: string;
+    color?: string;
+    onDelete?: Function;
+    className?: string;
+    theme?: ITheme;
+    styles?: IStyleFunctionOrObject<IChipStyleProps, IChipStyles>;
+}
+
+interface IChipStyleProps {
+    className?: string;
```

Comment:
Is this used?

Reply:
It was not, but should have been in the getStyles() function merging classes / styles. Would allow for adhoc classes, eg<Chip ... className={"some-class"} ... /> & the resulting HTML <div class="chip-container some-class ...">...</div>